fab/services/templates/supervisor_formsplayer_spring.conf

```diff
@@ -0,0 +1,8 @@
+[program:{{ project }}-{{ environment }}-formsplayer-spring]
+command=java -jar {{ code_current}}/submodules/formsplayer/build/libs/formsplayer.jar
+user={{ sudo_user }}
+autostart=true
+autorestart=true
+stdout_logfile={{ log_dir }}/formsplayer-spring.log
+redirect_stderr=true
+stderr_logfile={{ log_dir }}/formsplayer-spring.error.log
```

Comment:
Should probably add `startsecs=10` or something bigger than the default of `1`.

Probably also need to specify some more command line options like server port or config file location.